File: pkg/pwrgrep/testdata/fixtures/csharp/Calls.cs
public class Calls
{
    void F()
    {
        // ruleid: cs-calls
        Console.WriteLine("hi");
        // ruleid: cs-calls
        Helper();
        // ruleid: cs-calls
        var o = new StringBuilder();
        // ok: cs-calls
        int x = 1 + 2;
    }

    void Helper() { }
}

File: pkg/pwrgrep/testdata/fixtures/java/Calls.java
public class Calls {
    void f() {
        // ruleid: java-calls
        System.out.println("hi");
        // ruleid: java-calls
        helper();
        // ruleid: java-calls
        Object o = new Object();
        // ok: java-calls
        int x = 1 + 2;
    }

    void helper() {}
}
